paper-input-container.html

```diff
@@ -311,7 +311,8 @@
     },
 
     listeners: {
-      'addon-attached': '_onAddonAttached'
+      'addon-attached': '_onAddonAttached',
+      'iron-input-validity-changed': '_onInputValidityChanged'
```

Comment:
Maybe call the event `iron-input-validate`? Properties that are `notify:true` fires a  `*-changed` event, and might be confusing.